lib/postgres_to_redshift/tuning.rb

```diff
@@ -0,0 +1,60 @@
+module PostgresToRedshift
+  class Tuning
+    DEFAULT_SORT_KEYS = %w[created_at] # We often filter by these columns
+
+    def initialize(table:, source_connection:)
+      @table = table
+      @table_name = table.target_table_name.downcase
+      @optimised_for_table_name = ENV['POSTGRES_TO_REDSHIFT_OPTIMISED_FOR_TABLE']&.downcase&.strip || table_name
```

Comment:
What will be the content of this env variable? Just to have an idea of its structure

Reply:
Just wondering if we will have to run a specific full import to optimise a table?

Reply:
Also, tiny suggestion, I think you can require `active_support/core_ext` and then do a `&.presence`, just in case the environment variable is present but blank? Not so sure if this matters later on in the code, if not, discard 😬

Reply:
It will be a table name. For example, we've currently got this set to `users`

Reply:
Fair point! I checked the code, and it will work if the ENV is blank as-is 👍